frontend/src/app/projects/models/model-description/model-description.component.html

```diff
@@ -62,7 +73,14 @@
             Delete
           </button>
         </span>
-        <button mat-flat-button color="primary" type="submit">Submit</button>
+        <button
```

Comment:
There is still a small bug in the handling of the backend error if the slug already exists: 

https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/cf40a2cf-f518-488d-b8c2-d6edea880199